code-challenges.test.js

```diff
@@ -13,20 +13,52 @@
 // --------------------1) Create a function that takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence.
 
 // a) Create a test with expect statements for each of the variables provided.
+describe("fiboNacci", () => {
+  it("takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence", () => {
+    const fibLength1 = 6
+    expectedOutput1= [1, 1, 2, 3, 5, 8]
 
-const fibLength1 = 6
-// Expected output: [1, 1, 2, 3, 5, 8]
+    const fibLength2 = 10
+    expectedOutput2= [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+    expect(fiboNacci(fibLength1)).toEqual(expectedOutput1)
+    expect(fiboNacci(fibLength2)).toEqual(expectedOutput2)
+  })
+})
 
-const fibLength2 = 10
-// Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+//ReferenceError: fiboNacci is not defined
 
 // b) Create the function that makes the test pass.
 
+  //create a function called fiboNacci
+    //look up the fibonacci sequence 0, 1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89, 144, 233, 377, 610, 987, 1597, 2584, 4181, 6765, 10946, 17711, 28657, 46368, 75025, 121393, 196418, 317811. It takes the last two numbers in the sequence and adds both to create the next number.
+    //Create a parameter called number
+      //the function will check that the argument is greater than two using an if statement
+      //if the number is not, it will return enter a number greater than two.
+      //if the argument is greater than two then the function will return the length of the fibonacci sequence according to the number using slice built in method.
+  //run jest to verify function
+
+  const fiboNacci = (number) => {
+    let fiboSeq =[1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89, 144, 233, 377, 610, 987, 1597, 2584, 4181, 6765, 10946, 17711, 28657, 46368, 75025, 121393, 196418, 317811]
```

Comment:
This does a good job in getting the tests to pass and covering the first 28 instances, however, what if the number provided is larger than 28?  We will want to consider ways to keep the code dynamic for all instances.  To do this, consider using iteration.